foundation.e.easy-installer.yml

```diff
@@ -0,0 +1,30 @@
+app-id: foundation.e.easy-installer
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: easy-installer
+
+finish-args:
+  - "--env=SNAP_NAME=easy-installer"
+  - "--env=SNAP_USER_COMMON=/var/tmp"
```

Comment:
SNAP? This is Flatpak.

Reply:
We use env vars SNAP_NAME and SNAP_USER_COMMON in the upstream project 
https://gitlab.e.foundation/e/devices/easy-installer/-/blob/master/src/main/java/ecorp/easy/installer/AppConstants.java#L220
will see so we can change it.

Reply:
You should follow XDG standards insteads.

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
also detecting if you are inside flatpak at runtime is done by checking for `/.flatpak-info`